src/main/java/frc/robot/subsystems/conveyor/commands/Feed.java

```diff
@@ -0,0 +1,34 @@
+package frc.robot.subsystems.conveyor.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.conveyor.Conveyor;
+
+public class Feed extends CommandBase {
```

Comment:
This command should not end it just opens the flap and gives positive power to the conveyor